cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -95,8 +95,11 @@ func NewOVFImporter(params *ovfimportparams.OVFImportParams) (*OVFImporter, erro
 	}
 	tarGcsExtractor := storageutils.NewTarGcsExtractor(ctx, storageClient, logger)
 	buildID := os.Getenv("BUILD_ID")
+
 	if buildID == "" {
```

Comment:
why does ovf importer use a random ID as fallback while others don't?

Reply:
IIRC, it was for testing resource labelling when run from local.